themes/build.js

```diff
@@ -0,0 +1,96 @@
+const request = require("request");
+const less = require("less");
+const fs = require("fs");
+const { variants, labels } = require("@catppuccin/palette");
+
+const themes = ["latte", "frappe", "macchiato", "mocha"];
+
+const url = "https://raw.githubusercontent.com/catppuccin/palette/main/less/";
+
+fs.mkdirSync("built");
```

Comment:
What should we do if this fails?

Reply:
I updated the mkdirs to safely handle themselves.